deploy/crds/Keycloak.yaml

```diff
@@ -29,6 +29,28 @@ spec:
           type: object
         spec:
           type: object
+          properties:
+            externalDatabaseSecret:
+              description: 'Opotional. A Secret containing the connection string and credentials to an external database. If not provided a Postgres
+                instance will be created in the same namespace'
+              type: string
+            adminCredentialSecret:
+              description: 'Optional. The name of the Secret that contains the admin password in a `password` property. If not provided a password will be
+                auto generated and written to a Secret with the name `admin-credentials`'
+              type: string
+              required: false
+            extensions:
+              description: 'A list of extensions to be loaded by Keycloak on startup'
+              type: array
+              items:
+                type: object
+                properties:
+                  name:
+                    description: 'Extension name'
+                    type: string
+                  version:
+                    description: 'Extension version'
+                    type: string
         status:
```

Comment:
I think we should also have a number of instances (since we'll be running HA configuration by default).

Reply:
Done.